docs/Home.md

```diff
@@ -119,6 +119,17 @@ The following permissions will grant you enough permissions to deploy to your cl
     ```bash
     kubectl create -f PATH_TO_YOUR_ROLE_BINDING_YAML
     ```
+##### Automation Option
+For the more restrictive permissions option, we offer an example automated solution for configuring RBAC permissions.
+1. Create your service account with a custom IAM role using the following terraform [file](rbac/example.tf) and run:
+    ```bash
+    terraform apply #assuming terraform init was already run
```

Comment:
I would add a note about ensuring the user's working directory contains the Terraform plan file.

Reply:
Good point. Done.